lib/ssl/src/tls_v1.erl

```diff
@@ -910,16 +910,14 @@ default_signature_algs(_) ->
 default_pre_1_3_signature_algs_only() ->
     Default = [%% SHA2
                {sha512, ecdsa},
-               {sha512, rsa},
```

Comment:
@IngelaAndin Sorry to revive this old PR. Is the intention that the `{sha256, rsa}` signature algorithm is still supposed to work with TLS 1.2 with the default list of signature algorithms (when both TLS 1.3 and 1.2 are supported), because the default list contains the `rsa_pkcs1_sha256` TLS 1.3 legacy scheme?

I am trying to understand how it's supposed to work, since I have observed issues with `SERVER ALERT: Fatal - Insufficient Security - :no_suitable_ciphers` after upgrading to OTP 26, since my `ssl` server no longer supports `{sha256, rsa}` by default. It isn't clear to me if this is intended to be changed.